test_functions/TestAnimations2D.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio
+** File description:
+** TestAnimations2D
+*/
+
+#include <raylib.h>
+#include "Texture2D.hpp"
+
+#define MAX_FRAME_SPEED     15
+#define MIN_FRAME_SPEED      1
+
+void TestAnimations2D()
```

Comment:
I suggest doing a class, or methode in texture class at least, Animator.
The goal is to automate the process hardcoded here about moving the rect each frame ect